src/sdp/SDP.ts

```diff
@@ -5,66 +5,567 @@ import SDPCandidateType from './SDPCandidateType';
 import SDPMediaSection from './SDPMediaSection';
 
 /**
- * [[SDP]] manages and helps munge an SDP during negotiation.
+ * [[SDP]] includes a few helper functions for parsing sdp string.
  */
-export default interface SDP {
+export default class SDP {
```

Comment:
Since we do not need to expose this to builders, can we just use the tag `/** @internal */` above this and do not use export keyword so we do not export it?

Reply:
I tried adding the tag `/** @internal */` and it started throwing errors as there are some external api files that make use of the SDP class.